EvidenceApi.Tests/V1/UseCase/CreateAuditUseCaseTests.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using AutoFixture;
+using EvidenceApi.V1.Boundary.Request;
+using EvidenceApi.V1.Domain;
+using EvidenceApi.V1.Gateways.Interfaces;
+using EvidenceApi.V1.UseCase;
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+
+namespace EvidenceApi.Tests.V1.UseCase
+{
+    public class CreateAuditUseCaseTests
+    {
+        private CreateAuditUseCase _classUnderTest;
+        private Mock<IEvidenceGateway> _evidenceGateway;
+        private readonly IFixture _fixture = new Fixture();
+        private AuditEventRequest _request;
+        private AuditEvent _created;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _evidenceGateway = new Mock<IEvidenceGateway>();
+            _classUnderTest = new CreateAuditUseCase(_evidenceGateway.Object);
+
+        }
+
+        [Test]
+        public void ReturnsTheCreatedAuditEvent()
+        {
+            SetupMocks();
+            var result = _classUnderTest.Execute(_request);
+            Console.WriteLine(_request.Path);
+            Console.WriteLine(_created.UrlVisited);
+            Console.WriteLine(result.UrlVisited);
+
+            //result.Id.Should().NotBeEmpty();
+            result.UrlVisited.Should().Be(_created.UrlVisited);
+            result.UserEmail.Should().Be(_created.UserEmail);
+            result.RequestBody.Should().Be(_created.RequestBody);
+            result.HttpMethod.Should().Be(_created.HttpMethod);
+        }
+
+        [Test]
+        public void CallsGatewaysUsingCorrectParameters()
+        {
+            SetupMocks();
+
+            _classUnderTest.Execute(_request);
+
+            _evidenceGateway.Verify(x => x.CreateAuditEvent(
+                It.Is<AuditEvent>(e =>
+                    e.UrlVisited == _request.Path &&
+                    e.UserEmail == _request.UserEmail &&
+                    e.HttpMethod == _request.Method &&
+                    e.RequestBody == _request.Request
+                )
+            ));
+        }
+
+        private void SetupMocks()
```

Comment:
You could move this to `SetUp`

Reply:
I can move the call to the `SetupMocks()` function to the `SetUp` but I would leave the implementation in a separate method to be consistent with other tests